learning-another-language/semicolons.md

```diff
@@ -0,0 +1,117 @@
+# Semicolons
+
+## Introduction: To Use or Not Use
+
+In JavaScript, semicolons are used to mark the end of a statement:
+
+```js
+console.log('Do I need to put a semicolon at the end of this to work?');
+```
+
+Officially, JavaScript expects semicolons to end the following kinds of statements:
+
+- variable assignment
+- expressions (such as `1 + 2;`)
+- do-while blocks
+- `continue` statements
+- `break` statements
+- `return` statements
+- `throw` statements to throw errors
```

Comment:
seems a little odd only this has extra detail, but maybe it's okay as a parenthetical?
```suggestion
- `throw` statements (used to throw, or raise, errors)
```